src/pages/CoinPage/CoinPage.tsx

```diff
@@ -1,113 +1,90 @@
+                  {store.coin.name}
                 </h1>
                 <span className={styles["coin-page__header__name-box__symbol"]}>
-                  ({coin.symbol.toUpperCase()})
+                  ({store.coin.symbol.toUpperCase()})
                 </span>
               </div>
             </div>
             <div className={styles["coin-page__header__price-info"]}>
               <span className={styles["coin-page__header__price-info__price"]}>
-                {`$${coin.market_data.current_price.usd}`}
+                {`$${store.coin.currentPrice.usd}`}
               </span>
               <span
                 className={classNames(
                   styles["coin-page__header__price-info__price-change"],
                   {
                     [styles.success]:
-                      coin.market_data.price_change_24h_in_currency.usd > 0,
+                      store.coin.priceChange24hInCurrency.usd > 0,
                     [styles.danger]:
-                      coin.market_data.price_change_24h_in_currency.usd < 0,
+                      store.coin.priceChange24hInCurrency.usd < 0,
                   }
                 )}
               >
                 {`${rounding(
-                  coin.market_data.price_change_24h_in_currency.usd,
+                  store.coin.priceChange24hInCurrency.usd,
                   6
                 )} (${rounding(
-                  coin.market_data.price_change_percentage_24h_in_currency.usd
+                  store.coin.priceChangePercentage24hInCurrency.usd
                 )}%)`}
               </span>
             </div>
           </div>
           <div className={classNames(styles["coin_page__chart"])}>
-            {chart ? (
+            {store.chart ? (
               <Chart
                 onMouseEvent={true}
-                coinData={chart.map((arr) => arr[2])}
-                coinLabels={chart.map((arr) => convertTimestamp(arr[0]))}
+                coinData={toJS(store.chart).map((arr) => arr[2])}
+                coinLabels={toJS(store.chart).map((arr) =>
+                  convertTimestamp(arr[0])
+                )}
               />
             ) : (
               <Loader />
             )}
           </div>
           <Card
             currency={"USD"}
-            key={coin.id}
-            name={coin.name}
-            subtitle={coin.symbol}
-            image={coin.image.thumb}
-            price={coin.market_data.current_price.usd}
-            priceChange={
-              coin.market_data.price_change_percentage_24h_in_currency.usd
-            }
+            key={store.coin.id}
+            name={store.coin.name}
+            subtitle={store.coin.symbol}
+            image={store.coin.image.thumb}
+            price={store.coin.currentPrice.usd}
+            priceChange={store.coin.priceChangePercentage24hInCurrency.usd}
             className={classNames(styles["coin_page__card"])}
             withChart={false}
             userType={true}
```

Comment:
пропсы `boolean` типа со значением `true` можно прокидывать вот так:
```suggestion
            userType
```